src/components/shopping/Shopping.tsx

```diff
@@ -0,0 +1,55 @@
+import { useContext, useEffect, useState } from 'react';
+import { GlobalContext } from '../../reducer';
+import { ShoppingProduct } from '../../types/shoppingProduct';
+import { Way } from '../way/Way';
+import { ShoppingCard } from './cardPrice/CardPrice';
+
+import './shopping.scss';
+
+export const Shopping = () => {
+  const [state] = useContext(GlobalContext);
+  const [totalCost, setTotalCost] = useState(0);
+
+  const getPrice = (product:ShoppingProduct) => {
+    return product.item.price - (
+      product.item.price / 100) * product.item.discount;
+  };
+
+  useEffect(() => {
+    setTotalCost(state.basketList.map((el:ShoppingProduct) => {
+      return getPrice(el) * el.value;
+    }).reduce((acc, el) => acc + el, 0));
+  }, [state.basketList]);
+
+  return (
+    <div className="wrapper-shopping">
+      <Way />
+      <h1>Cart</h1>
+      <div className="main-info-shopping">
+        {state.basketList.length
+          ? (
+            <>
+              <div className="wrapper-shopping-cards">
+                {
+                  state.basketList.map((product: ShoppingProduct) => {
+                    return (
+                      <ShoppingCard product={product} key={product.item.age} />
+                    );
+                  })
```

Comment:
use next syntax when you return at once in arrow functions
```suggestion
                  state.basketList.map((product: ShoppingProduct) => (
                     <ShoppingCard product={product} key={product.item.age} />
                   ))
```

Reply:
+